### PR TITLE
Kim mb 304 legalese redux refactor

### PR DESCRIPTION
**Pull request by kimallen**
## Description

Refactors transition from legalese page to PPM submission.  Replaces `signAndSubmitForApproval` in the `Legalese/ducks.js` file to use a combo of `createSignedCertificate` and `submitMoveForApproval` and make the info available in entities.

To show the success banner, I added a temporary action/reducer (will be refactored to use a global flash message in a new pattern in [this story](https://dp3.atlassian.net/browse/MB-2978)).

Removes unused functions and components (`SubmitPpm.jsx`, `PaymentReview.jsx`).  ([discussion](https://ustcdp3.slack.com/archives/CTNTFJSBA/p1591995271033900) about PaymentReview.jsx)

## Reviewer Notes

The payment request submission flow already used entities, so didn't need refactoring.

I removed the `signAndSubmitPpm` which didn't seem to be used besides the `SubmitPpm.jsx`, which we didn't seem to need (I _think_ based on the ticket that it was previously used for the payment request submission before the refactor).  That function did use a different handler (`SubmitPersonallyProcuredMoveHandler` in `personally_procured_move.go`) than `signAndSubmitForApproval` (`SubmitMoveHandler` in `moves.go`).  I chose to use the `moves.go` handlers because it updates the status of both the move and the ppm to SUBMITTED.  The other handler only updates the PPM.  

Created [another story](https://dp3.atlassian.net/browse/MB-2967) that includes exploring whether one of the handlers can be removed (or if it should be used in combo with a refactored handler)

## Setup

Walk through the PPM flow through submission of a PPM (I used user `profile@co.mple.te`).  You should see a success banner when redirected to the landing page.  Make sure that the legalese text changes based on whether they have storage or not (can edit in the Review page).

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-304) and [other submitPpm Jira story](https://dp3.atlassian.net/browse/MB-303) for this change

### CODE REVIEW

File: src/scenes/Legalese/ducks.js [hunk 1]
```diff
@@ -1,150 +1,35 @@
-import { GetCertificationText, CreateCertification } from './api.js';
```
Comment (sojeri) [inline src/scenes/Legalese/ducks.js:1]: 🦆? what does this file name mean?

File: src/scenes/Legalese/index.jsx [hunk 2]
```diff
@@ -25,52 +28,66 @@ export class SignedCertification extends Component {
...
+    if (hasRequestedAdvance) txt.push(ppmAdvance);
+    txt.push(additionalInformation);
+    return txt.join('');
   }
 
+  submitCertificate = () => {
+    const signatureTime = moment().format();
+    const { currentPpm, moveId } = this.props;
+    const certificate = {
+      certification_text: this.getCertificationText(currentPpm.has_sit, currentPpm.has_requested_advance),
```
Comment (sojeri) [inline src/scenes/Legalese/index.jsx:46]: what does has_sit mean?

Comment (kimallen) [reply src/scenes/Legalese/index.jsx:46]: sit is "storage in transit".  So they've opted for  putting stuff in storage

File: src/scenes/Legalese/ducks.js (hunk 1 above)
Comment (kimallen) [reply src/scenes/Legalese/ducks.js:1]: "Legalese" refers to the language in the disclaimer that the service member signs for (certification).  Doesn't really fit anymore. This file will get deleted once a flash message system refactor is set up.

File: src/scenes/Legalese/index.jsx (hunk 2 above)
Comment (kimallen) [reply src/scenes/Legalese/index.jsx:46]: They do that on the "Dates and Locations" page.  It asks if they will put anything in storage.  When they select yes, it asks number of days.

**Pull request merged**